language-design.md

```diff
@@ -59,7 +83,13 @@ What does the post on grayscale tell us about the process of API design?
 
 **Response**
 
+Lea Verou’s forum post on grayscale is the embodiment of several important design paradigms explored in Bloch’s API design paper. To elaborate, this forum post was created by Lea Verou to ask users to vote on the best name for an API function she was designing. Specifically, the function name was supposed to describe a routine that sets a color to an appropriate shade of gray when provided a lightness and optional alpha parameter.
+ 
+By asking her users to vote on a function name, Verou is clearly in compliance with Bloch’s suggestion that “Names matter,” and “API design is not a solitary activity” [Bloch, 2006, 506]. For instance, if the function names or her users did not matter, why would the developer ever go through the trouble of poling users for something so trivial? It is because, like a DSL, a good API strives to be a simplification of functionality that is intuitive for users. Therefore, a good API should not be what is easiest for the designer to implement and remember, but should conform to the needs of the user base.
```

Comment:
This is a great place to quote Bloch. It's a great idea to combine the different readings and show how the ideas of good API design can be applied here.